src/components/BuySell/components/BuySellButton.tsx

```diff
@@ -89,15 +87,6 @@ const BuySellButton: React.FC = () => {
     buySellToken.toLowerCase() === 'btcfli' &&
     btc2xfliApproval.isApproving
 
-  const cgiApprovalRequired =
```

Comment:
We'll still need to enable CGI approval to the uniswap router to allow users to Sell